src/components/FisheriesMap.vue

```diff
@@ -40,25 +151,65 @@ export default {
         format: 'image/png',
         transparent: true,
         attribution: 'USGS',
+        baseLayer: true,
       }
     )
-    var center = [61.668562, -163.916567]
     this.map = L.map('map', {
-      minZoom: 4.5,
+      minZoom: 4,
       maxZoom: 8,
+      zoomSnap: 0.1,
       scrollWheelZoom: false,
       layers: [baseLayer],
-    }).setView(center, 4.5)
+    })
+
+    this.enteredString = this.searchString
   },
   created() {
+    this.$store.dispatch('fetchAccess')
```

Comment:
It'd be lovely to only need a single dispatch to get all info, instead of all these different API requests.